molgenis/get_utils.py

```diff
@@ -0,0 +1,64 @@
+from typing import List, Optional
```

Comment:
Shall we rename to `query_utils.py`? It's a little more descriptive

Reply:
Good suggestion!